include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -263,6 +273,11 @@ SECTIONS
 
     GROUP_START(RAMABLE_REGION)
 
+	/* Align the start of image SRAM with the
+	 * minimum granularity required by MPU.
+	 */
+	. = ALIGN(_region_min_align);
+	_image_ram_start = .;
```

Comment:
_image_ram_start is, now, placed in the right point

Reply:
I thought we were going to get rid of it.

What is it for?

Reply:
I am going to need this very soon, in ARMv8-M to mark the area from the beginning of SRAM until the first MPU region that corresponds to some kernel feature (e.g. Application Memory). Looks like there might be some SOC-specific linker sections there.